public/app/features/alerting/unified/Analytics.test.ts

```diff
@@ -0,0 +1,42 @@
+import { dateTime } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+
+import { isNewUser } from './Analytics';
+
+jest.mock('@grafana/runtime', () => ({
+  getBackendSrv: jest.fn().mockReturnValue({
+    get: jest.fn(),
+  }),
+}));
+
+describe('isNewUser', function () {
+  it('should return true if the user has been created within the last two weeks', async () => {
+    const newUser = {
+      id: 1,
+      createdAt: dateTime().subtract(14, 'days'),
+    };
+
+    getBackendSrv().get = jest.fn().mockResolvedValue(newUser);
+    const getRequestSpy = jest.spyOn(getBackendSrv(), 'get');
```

Comment:
This spy is not necessary as you have mocked the `getBackendSrv().get`, you can check the following tests like that:
`expect(getBackendSrv().get).toHaveBeenCalledTimes(1);`